src/dabplus-on-air-processing.liq

```diff
@@ -29,11 +29,15 @@ input =
   end
 
 output =
-  if enable_processor then
-    log("Audio Processor enabled", label="dabplus")
+  if processor == "mk" then
```

Comment:
I haven't tested if %include takes strings as an argument.

It's all a bit hacky, I'll do a "real" refactor when/if we decide to ditch mk_liquidsoap_processing.

Right now the input/output "api" that these processor files support where the easy way to use the mk script without forking.

When I revisit this next, I'll probably refactor individual processors into functions with a proper non-include based way to call them. I don't really like how they get executed as "side effect" when they are included.

Let's get it to sound like we want it to and then look at what remains before refactoring this pattern.